extensions/amp-timeago/0.1/amp-timeago.js

```diff
@@ -67,6 +67,15 @@ export class AmpTimeAgo extends AMP.BaseElement {
     }
   }
 
+  /** @override */
+  mutatedAttributesCallback(mutations) {
+    const datetime = mutations['datetime'];
+    if (datetime !== undefined) {
+      this.datetime_ = this.element.getAttribute('datetime');
```

Comment:
I believe this can just be `datetime` cost per above, not need to read it again.

Reply:
Good point! I've now updated that.